addons/account/models/account_journal_dashboard.py

```diff
@@ -278,9 +277,7 @@ def get_journal_dashboard_datas(self):
                 SELECT COUNT(st_line.id)
                 FROM account_bank_statement_line st_line
                 JOIN account_move st_line_move ON st_line_move.id = st_line.move_id
-                JOIN account_bank_statement st ON st_line.statement_id = st.id
                 WHERE st_line_move.journal_id IN %s
-                AND st.state = 'posted'
```

Comment:
So there is no validation that a line is ready to be reconciled anymore?

Reply:
Lines are auto-posted now, at least at the moment.